src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt

```diff
@@ -628,8 +628,11 @@ class MailboxSceneController(private val scene: MailboxScene,
             changeMode(multiSelectON = false, silent = false)
             when(result) {
                 is MailboxResult.MoveEmailThread.Success ->  {
-                    reloadMailboxThreads()
```

Comment:
I think you should call `scene.setToolbarNumberOfEmails(getTotalUnreadThreads())`